language/diem-vm/src/diem_vm.rs

```diff
@@ -37,7 +37,6 @@ use move_vm_types::{
     values::Value,
 };
 use std::{convert::TryFrom, sync::Arc};
-use vm::errors::Location;
```

Comment:
I don't know where this change came from. It wasn't in my #6780 and regardless it doesn't work since there is still a reference to Location::Undefined in this file.